packages/astro/src/@types/astro.ts

```diff
@@ -13,8 +21,35 @@ export interface AstroComponentMetadata {
 }
 
 /**
- * The Astro User Config Format:
- * This is the type interface for your astro.config.mjs default export.
+ * Astro.* available in all components
+ * Docs: https://docs.astro.build/reference/api-reference/#astro-global
```

Comment:
![Screen Shot 2021-11-12 at 17 40 16](https://user-images.githubusercontent.com/1369770/141598730-fc9f03b1-3591-484e-afb6-fdb95a1f58b5.png)

Adding a URL makes finding the API docs easier